robottelo/constants/__init__.py

```diff
@@ -394,6 +395,15 @@
         'distro': 'rhel7',
         'key': 'rhs',
     },
+    'rhsc7': {
+        'id': 'rhel-7-server-satellite-capsule-6.11-rpms',
+        'name': ('Red Hat Satellite Capsule 6.11 for RHEL 7 Server RPMs x86_64'),
```

Comment:
Just curious to know, why do we need old 6.11 repo? and can't we bump it to latest 6.16? same for rhsc8..

Reply:
TBH, I just moved it so the `rhscX` repos are sorted from 7 to 9. Haven't touched the values of `rhsc7`.

Reply:
yes, I understand but I feel we can leverage the opportunity to bump this versions as well based on its usage in the repo